.github/workflows/static-checks.yaml

```diff
@@ -107,6 +107,9 @@ jobs:
           echo "Set environment variables for the libseccomp crate to link the libseccomp library statically"
           echo "LIBSECCOMP_LINK_TYPE=static" >> $GITHUB_ENV
           echo "LIBSECCOMP_LIB_PATH=${libseccomp_install_dir}/lib" >> $GITHUB_ENV
+      - name: Install protobuf-compiler
+        if: ${{ matrix.command != 'make vendor' && ( matrix.component == 'agent' || matrix.component == 'runk' ) }}
+        run: sudo apt-get -y install protobuf-compiler
```

Comment:
Why do we need this?

Reply:
The package `protobuf-compiler` is required to build `image-rs`. If it is not installed on the host machine, an error will occur when running the `make checks` for the agent: https://github.com/kata-containers/kata-containers/actions/runs/7109227045/job/19353813108?pr=8484